shapeshift/web
# feat: parse send to vanity/address/url in global search

## Description

This PR:

- Adds support for a new "Action" category/rows in the global search
- When inputting/pasting a valid vanity/address or URL in the global search, the matching native asset will appear as an option for sending to that address

## Pull Request Type


## Issue (if applicable)

N/A

## Risk



This could regress send/QR codes parsing given the change in input parsing, test accordingly

## Testing



- Paste an address/URL/vanity address in the global search
- Ensure "Send to" shows up as an action for a valid ns address (e.g vitalik.eth), addresses (on any supported chain), and Ethereum/UTXO payment URLs 

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)

<img width="550" alt="image" src="https://github.com/shapeshift/web/assets/17035424/2f46ebaf-347f-4e3f-b20a-1b3b068110eb">
<img width="550" alt="image" src="https://github.com/shapeshift/web/assets/17035424/ff56e1c2-4ee4-434f-87f3-f918bc3ed407">
<img width="538" alt="image" src="https://github.com/shapeshift/web/assets/17035424/31d46374-c23e-4306-b47a-10e59918d02a">
